web/modules/custom/rsvplist/config/schema/rsvplist.schema.yml

```diff
@@ -0,0 +1,10 @@
+rsvplist.settings:
+  type: config_object
+  label: RSVP List Content Type Settings
+  mapping:
+    allowed_types:
+      type: sequence
+      label: Content Types the RSVP Form Can Display On.
```

Comment:
Label value must be written within single quotation marks. The same is applied to the next label.

Reply:
@Vishalprasad28 ?